lab3/main.py

```diff
@@ -0,0 +1,71 @@
+
+def max_equal(arr, n):
+    maxLen = Len = 1
+
+    for i in range(1, len(arr)):
+        if arr[i] == arr[i-1]:
+            Len = Len + 1
```

Comment:
i = i + 1 reassigns i, i += 1 increments i by 1

it's better if you use += in this case